src/Test.UnitTests.Sarif/Visitors/InsertOptionalDataVisitorTests.cs

```diff
@@ -26,22 +27,32 @@ public InsertOptionalDataVisitorTests(ITestOutputHelper outputHelper, DeletesOut
 
         protected override string ConstructTestOutputFromInputResource(string inputResourceName, object parameter)
         {
-            PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(
+            SarifLog actualLog = PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(
                 GetResourceText(inputResourceName),
                 formatting: Formatting.Indented,
-                out string transformedLog);
+                updatedLog: out _);
+
+            // Some of the tests operate on SARIF files that mention the absolute path of the file
+            // that was "analyzed" (InsertOptionalDataVisitor.txt). That path depends on the repo
+            // root, and so can vary depending on the machine where the tests are run. To avoid
+            // this problem, both the input files and the expected output files contain a fixed
+            // string "ENLISTMENT_ROOT" in place of the directory portion of the path. But some
```

Comment:
>ENLISTMENT_ROOT [](start = 23, length = 15)

Can you please think of a name that is an improvement on the previous (that meets the standard that prompted you to change it) but which still indicates that the value is rewritten by the test infrastructure? This is not obvious after the change you've made.